GettingStarted_4.0.0-rc1.md

```diff
@@ -0,0 +1,111 @@
+What’s New:
+----------
+
+   1. Unified code for IPv4 and IPv6
+   2. Multiple Interface/Endpoint support
+      (Reference: https://www.freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_TCP/freertostcp-multiple-interfaces.html)
+   3. Neighbor Discovery & Router Advertisement
+   4. New WinSim demo to support both IPv4 and IPv6
+
+FreeRTOS-Plus-TCP 4.0.0-rc1 complies with the following code code quality criteria:
+
+   1. 100% Unit test cases Passed for IPv4
+   2. IPv4 Protocol Testing with Maxwell Pro
+   3. All functions comply with the MISRA 2012 coding standard.
+   4. All functions passing Coverity static checking.
+   5. CBMC IPv4 memory safety proofs
+   6. GNU Complexity score same as of main branch ( less than 8 *with exceptions)
+
+
+Moving to 4.0.0-rc1 from 3.0.0:
+-----------------------------
+In version 4.0.0-rc1, new files have been added to support IPv6 functionality and each file has been broken down into logically seperated IPv4 and IPv6 files. The folder structure of FreeRTOS-Plus-TCP has not changed.
+
+Some of the APIs have changed which is illustrated in the section below. However, there is a backward compatibility mode provided as well.
+
+
+Running Demos:
+-------------
+
+The demo branch compatible with this branch:
+https://github.com/FreeRTOS/FreeRTOS/tree/devIPv6
+
+In all the demos, there is a backward compatibility mode which can be enabled by setting the flag “ipconfigIPv4_BACKWARD_COMPATIBLE” to 1 in the header file “FreeRTOSIPConfigDefaults.h”.
+
+
+This flag is by default set to zero.
+
+New API's in 4.0.0-rc1:
+----------------------
+Change 1:
+
+   1. Old API: FreeRTOS_IPInit
+   2. New API: FreeRTOS_IPInit_Multi
+   3. Backward Compatibility - Yes
+   4. Change:
+      1. FreeRTOS_IPInit - Backward compatibility with the IPv4 FreeRTOS+TCP main branch which only supports single network interface. Can still be used for single  interface IPv4 systems.
```

Comment:
Can still be used for single  interface IPv4 systems. ==> "It" is missing at start of sentence.